executor/aggfuncs/func_max_min.go

```diff
@@ -111,16 +192,46 @@ func (e *maxMin4Int) UpdatePartialResult(sctx sessionctx.Context, rowsInGroup []
 		if isNull {
 			continue
 		}
-		if p.isNull {
-			p.val = input
-			p.isNull = false
+		p.Append(input)
```

Comment:
This may cause more time for a function with an aggregation use. Aggregation just calculate once so that we do not need to construct this heap.

Reply:
The  sliding window is rely on `UpdatePartialResult()` to initiate.  Maybe we have to construct this heap.